tools/scripts/build/templateInstallationPackage/autonomiccsJars/moveJars.sh

```diff
@@ -23,10 +23,24 @@
 # Autonomiccs jars
 cp autonomiccsJars/autonomic*.jar /usr/share/cloudstack-management/webapps/client/WEB-INF/lib/
 cp autonomiccsJars/starthost-plugin*.jar /usr/share/cloudstack-management/webapps/client/WEB-INF/lib/
-# Do not change this line, we remove the '.jar' extension on purpose.
-wakeOnLanNewFileName=$(ls autonomiccsJars/wakeonlan-service-*.jar) && wakeOnLanNewFileName=${wakeOnLanNewFileName##*/} && wakeOnLanNewFileName=${wakeOnLanNewFileName::-4}
-cp autonomiccsJars/wakeonlan-service-*.jar "/usr/share/cloudstack-management/webapps/client/WEB-INF/lib/$wakeOnLanNewFileName";
 
 # Autonomiccs platform dependencies.
 cp autonomiccsJars/dependencies/*.jar /usr/share/cloudstack-management/webapps/client/WEB-INF/lib/
 
+# Creation of a folder to hold the jar files that needs to get transfered to VMs.
+baseAutonomiccsJarFolder="/var/lib/autonomiccs/jars/";
+
+jadeAgentsFolder="${baseAutonomiccsJarFolder}jade/";
```

Comment:
at this line and the next you do not need ${}, you can just use `$baseAutonomiccsJarFolder`